### ISSUE TITLE
Activate HTTPS.

### ISSUE THREAD

**Issue by davidbonachera** [nickolasclarke/juke#6]
> The https:// and tor:// gateways are also unavailble upon redeployment, which contains the client configuration details, mirrors, etc.

Hi,

To reconnect the https:// protocol we need to :
1. nano /etc/default/sslh and modify the last line with our new ip_v4_address
2. nano /etc/monit/monitrc and edit the sslh part with our new ip_v4_address
3. restart service sslh

Then you can get back to your https address. I think we can automate those steps maybe.

Meanwhile, i'm trying to adapt this project to Vultr API if you don't mind @nickolasclarke

**Comment by nickolasclarke** [comment]
Yep all easily modified automatically. Simply an adaptation on the
getShadowsocks method. Ideally this would probably end up being a refactor
that pulls the helper methods found in getShadowsocks out and probably into
their own module and then write modules for each service. Feel free to send
along a PR for any of this, including for vultr support.

**Comment by davidbonachera** [comment]
I'll try, but my ES6 skills are not really great. Stay tuned.

**Comment by nickolasclarke** [comment]
no worries mate, this was my first es6 project. First real complete project
really.
